yoti_python_sdk/activity_details.py

```diff
@@ -19,19 +20,23 @@ def __init__(self, receipt, decrypted_profile=None):
                     field.value,
                     field.content_type
                 )
-                self.user_profile[field.name] = value
+
+                anchors = Anchor().parse_anchors(field.anchors)
+
+                self.profile[field.name] = attribute.attribute(field.name, value, anchors)
```

Comment:
Is it possible to import `from yoti_python_sdk.attribute import attribute` so we don't have to repeat the name here?

Reply:
Yes that makes sense, will change